imgui-core/src/main/kotlin/imgui/static/dockNode.kt

```diff
@@ -1087,11 +1087,9 @@ fun dockNodePreviewDockRender(hostWindow: Window, hostNode: DockNode?, rootPaylo
     val isTransparentPayload = io.configDockingTransparentPayload
 
     // In case the two windows involved are on different viewports, we will draw the overlay on each of them.
-    var overlayDrawListsCount = 0
-    val overlayDrawLists = ArrayList<DrawList>()
-    overlayDrawLists[overlayDrawListsCount++] = getForegroundDrawList(hostWindow.viewport!!)
+    val overlayDrawLists = mutableListOf(getForegroundDrawList(hostWindow.viewport!!))
     if (hostWindow.viewport !== rootPayload.viewport && !isTransparentPayload)
-        overlayDrawLists[overlayDrawListsCount++] = getForegroundDrawList(rootPayload.viewport!!)
+        overlayDrawLists.add(getForegroundDrawList(rootPayload.viewport!!))
```

Comment:
Looks fine, would you just like to use the `+=` operator here? 

If not, I'll accept it anyway

Reply:
Sure.